FFCDemoPaymentService/FFCDemoPaymentService.csproj

```diff
@@ -1,23 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <LangVersion>latest</LangVersion>
-    <Version>5.0.12</Version>
+    <Version>6.0.0</Version>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.3.0"/>
-    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.1.2"/>
-    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.17.0"/>
-    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.HealthChecks" Version="2.2.0"/>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="5.0.5">
+    <PackageReference Include="Azure.Identity" Version="1.10.0" />
```

Comment:
1.10.2 ?

Reply:
Looks like version updated recently. will take it